docs/blog/2019-08-07-theme-jam/index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Gatsby Theme Jam: Congratulations Vojtěch Holík and Allan Pooley!"
+date: 2019-08-07T12:00:00
+author: Jason Lengstorf
+excerpt: >
+  During the Theme Jam, 111 themes wer created by our incredible community. Join us in celebrating the Gatsby community (that’s you!) and congratulating Vojtěch and Allan!
+tags:
+  - themes
+  - community
+---
+
+At the beginning of July, [Brent Jackson](https://twitter.com/jxnblk) joined _Learn With Jason_ to show [how to use `theme-ui`](https://youtu.be/6Z4p-qjnKCQ?list=PLz8Iz-Fnk_eTpvd49Sa77NiF8Uqq5Iykx). During the show, we announced the stable launch of [Gatsby themes](/docs/themes/) and kicked off the [Theme Jam](https://themejam.gatsbyjs.org), a celebration of our incredible community of creators.
+
+## What is the Theme Jam?
+
+In a nutshell, the Theme Jam was a way for Gatsby to give back to its community: everyone who published a theme and submitted it to the Theme Jam will receive exclusive swag designed by [Maggie Appleton](https://maggieappleton.com/), and two creators who published themes our team reviewed as exemplary are getting an all-expenses paid trip to [Gatsby Days](https://www.gatsbyjs.com/resources/gatsby-days/). (For more details, see the [rules](https://themejam.gatsbyjs.org/rules).)
```

Comment:
Maybe we could split this into two sentences?

Reply:
Good call. Done!